fastly.tf

```diff
@@ -1,5 +1,5 @@
 locals {
-  host     = regex("[^\\.]*", var.domain)
+  host     = regex("[^\\.]*", var.domains[0])
   backends = [for i in range(0, var.number_of_backends) : "backend_${i}"]
   director = var.director ? toset(formatlist(local.host)) : []
```

Comment:
Why just first? 
In this case maybe the definition of it changes. In a sense what is practical of choosing first for it ?I see its only the director Im just not 100% sure. maybe we need to slightly adapt doc?